app/controllers/v1/service_types_controller.rb

```diff
@@ -106,6 +106,39 @@ def destroy
     head :no_content
   end
 
+  # /service_types/update
+  # Update the service types from the catalog
+  def updateAll
+    require 'FileUtils'
+    require 'find'
+
+    if !current_actor.superadmin
+      json_response({ message: "You don't have permission to update service types" }, :unauthorized)
+      return
+    end
+
+    if (Dir["mastermind-services"]).length != 0 then
+      FileUtils.rm_rf('mastermind-services')
+    end
+    `git clone https://github.com/martel-innovate/MasterMind-Service-Catalog mastermind-services`
+
+    Find.find('mastermind-services') do |path|

```

Comment:
Will you add some tests for this new functionality?

Reply:
@taliaga It's a bit difficult to keep it consistent since I'm downloading an external repo. May try to separate this function and test it separately.